recipes/corsid/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "corsid" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 7bf90464e0b3f6d59b9cb0dd6a972fdd373ac2b7c468ee5769c8398d44245167
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python
+    - setuptools
+  host:
+    - numpy
+    - pandas
+    - pip
+    - pysam
+    - pytablewriter
+    - python
+    - tqdm
```

```suggestion
  host:
    - pip
    - python
```
a build section is not needed for python packages and only python and pip are needed in host

---

Thank you